src/ralph/cli.py

```diff
@@ -103,12 +104,33 @@ def fetch(backend, archive, **options):
     logger.info("Fetching archive %s from the configured %s backend", archive, backend)
     logger.debug("Backend parameters: %s", options)
 
+    options["fs_path"] = FS_STORAGE_DEFAULT_PATH
```

Comment:
You need to fix default value in the CLI argument instead. I've pushed a fix for this.